Web/starter_project_iconics/pages/blog/singleBlog.tsx

```diff
@@ -0,0 +1,80 @@
+
+// type Blog  = {
+//     title : string,
+
+
+// }
+
+// interface props {
+//     title : Blog.title,
+
+// };
+
+const singleBlog = ({}) => {
```

Comment:
let us stick to PascalCase for component names (and i think a better name for this might be blog detail instead of single blog)
and I think there is a specific way to define pages in Next other than just defining an empty function
try to check it out to clarify what is what